CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. More information can be found in each of the README.md as well as  https://docs.airswap.io/ about the individual repos.
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
+
+This project uses lerna to handle managing the mono-repo. Lerna is a tool for managing JavaScript projects with multiple packages.
+
+Run the below command from the root directory to start downloading necessary packages:
+
+`yarn install `
+
+
+## Running the tests
+A great way to explore the code base is to run the tests.
+
+We can run all tests with:
+
+`yarn test`
+
+
```

Comment:
If you want to run the tests of just 1 sub-repo, then run the same command inside the sub-repo.
e.g. `cd source/swap`, `yarn test`